<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test_10BClass</class>
 <widget class="QMainWindow" name="test_10BClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>test_10B</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="mytext"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文本</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>其它</string>
     </property>
     <addaction name="actionSetting"/>
     <addaction name="actionHelp"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionOpen"/>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>其它</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSetting"/>
   <addaction name="actionHelp"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="test_10B.qrc">
     <normaloff>:/test_10B/Resources/new.png</normaloff>:/test_10B/Resources/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="test_10B.qrc">
     <normaloff>:/test_10B/Resources/save.png</normaloff>:/test_10B/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="test_10B.qrc">
     <normaloff>:/test_10B/Resources/setting.png</normaloff>:/test_10B/Resources/setting.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="test_10B.qrc">
     <normaloff>:/test_10B/Resources/help.png</normaloff>:/test_10B/Resources/help.png</iconset>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="test_10B.qrc">
     <normaloff>:/test_10B/Resources/oepn.png</normaloff>:/test_10B/Resources/oepn.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="test_10B.qrc"/>
 </resources>
 <connections/>
</ui>
